diff --git a/node_modules/@sitecore-content-sdk/cli/dist/cjs/scripts/project/build.js b/node_modules/@sitecore-content-sdk/cli/dist/cjs/scripts/project/build.js
index de8d2cb..a5b689b 100644
--- a/node_modules/@sitecore-content-sdk/cli/dist/cjs/scripts/project/build.js
+++ b/node_modules/@sitecore-content-sdk/cli/dist/cjs/scripts/project/build.js
@@ -36,5 +36,7 @@ function handler(argv) {
                 yield command();
             }
         }
+
+        console.log('[TEST] All the commands have been executed');
     });
 }
