diff --git a/node_modules/@sitecore-content-sdk/cli/dist/cjs/scripts/project/build.js b/node_modules/@sitecore-content-sdk/cli/dist/cjs/scripts/project/build.js
index de8d2cb..bf60efe 100644
--- a/node_modules/@sitecore-content-sdk/cli/dist/cjs/scripts/project/build.js
+++ b/node_modules/@sitecore-content-sdk/cli/dist/cjs/scripts/project/build.js
@@ -36,5 +36,8 @@ function handler(argv) {
                 yield command();
             }
         }
+
+        // Exit the process to avoid hanging the process by custom build commands
+        process.exit(0);
     });
 }
